The Asset Manager is now visible (#330)
akeneo/pim-api-docs: content/concepts/products.md

# Products

> The Product is the central resource of our PIM, seems legit since we are doing Product Management. :wink:

Reviewer: ```suggestion
The Product is the central resource of our PIM, it makes sense right since we are doing Product Management. :wink:
```

Reviewer: Je trouve legit très informel

Author: ```suggestion
The **Product** is the central resource of our PIM and, when you think about it, it makes perfect sense since what we are doing is Product Management. :wink:
```

> In the sections below, you will find all the different flavours of products you can meet in the PIM.  

Reviewer: ```suggestion
In the sections below, you will find all the different flavors of products you can find in the PIM.  
```

Reviewer: not really sure about flavors!

> Each section contains an explanation of the concept behind the resource, what is its usage in the PIM and its JSON format in order to interact with the API.

Reviewer: ```suggestion
Each section below contains an explanation of the concept behind these resources. You will find out more about their usage in the PIM and their JSON format in order for them to interact with the API. 
```
